packages/firestore/src/local/simple_db.ts

```diff
@@ -132,6 +139,10 @@ export class SimpleDb {
 
   /** Returns true if IndexedDB is available in the current environment. */
   static isAvailable(): boolean {
+    // If IndexedDb is available directly (ex: in case of web workers).
+    if (typeof indexedDB !== 'undefined') {
+      return true;
+    }
     if (typeof window === 'undefined' || window.indexedDB == null) {
```

Comment:
Per above, since `window` is _usually_ the global object and we've already checked for the indexedDB global directly, is there a case where we will ever hit this?

Reply:
I think we just need to register the IndexedDb shim directly on `globalAny` in `node_persistence.ts`.

Reply:
Doesn't that mean that "if (typeof window === 'undefined' || window.indexedDB == null)" is now no longer needed? Is your distinction from your first response here still relevant? if so, we should probably talk this through offline.